DoctorWho.web/Controllers/EpisodeController.cs

```diff
@@ -2,29 +2,33 @@
 using DoctorWho.Db.Models;
 using DoctorWho.Db.Repositories;
 using DoctorWho.DTOs.Models;
+using DoctorWho.web.Validators;
 using DoctorWho.Web.Validators;
 using FluentValidation.Results;
 using Microsoft.AspNetCore.Mvc;
 
 namespace DoctorWho.web.Controllers
 {
-    [Route("/episodes")]
+    [Route("/episode")]
```

Comment:
So in the Other PR you use plural for your resources but now you changed to singular, make sure all your api routes follow one convention otherwise your API is more difficult to discover.